include/psa/crypto_values.h

```diff
@@ -2011,4 +2076,175 @@ static inline int mbedtls_svc_key_id_is_null( mbedtls_svc_key_id_t key )
+ */
+#define PSA_PAKE_SIDE_FIRST                ((psa_pake_side_t)0x0001)
+
+/** The second peer in a balanced PAKE.
+ *
+ * Although balanced PAKE protocols are symmetric, some of them needs an
+ * ordering of peers for the transcript calculations. If the protocol does not
+ * need this, either #PSA_PAKE_SIDE_FIRST or #PSA_PAKE_SIDE_SECOND are
+ * accepted.
+ */
+#define PSA_PAKE_SIDE_SECOND                ((psa_pake_side_t)0x0002)
+
+/** The client in an augmented PAKE.
+ *
+ * Augmented PAKE protocols need to differentiate between client and server.
+ */
+#define PSA_PAKE_SIDE_CLIENT                ((psa_pake_side_t)0x0101)
+
+/** The server in an augmented PAKE.
+ *
+ * Augmented PAKE protocols need to differentiate between client and server.
+ */
+#define PSA_PAKE_SIDE_SERVER                ((psa_pake_side_t)0x0102)
+
+/** The PAKE uses elliptic curves.
+ *
+ * The corresponding family type is ::psa_ecc_family_t. in determining a
+ * specific curve in the family ::psa_pake_bits_t values are interpreted in the
+ * exact same way as ::psa_key_bits_t would.
+ *
+ * Input and output during the operation can involve group elements and scalar
+ * values:
+ * -# The format for group elements is the same as for public keys on the
+ *  specific curve would be. For more information, consult the documentation of
+ *  psa_export_public_key().
+ * -# The format for scalars is the same as for private keys on the specific
+ *  curve would be. For more information, consult the documentation of
+ *  psa_export_key().
+ */
+#define PSA_PAKE_PRIMITIVE_TYPE_CURVE       ((psa_pake_primitive_type_t)0x01)
+
+/** The PAKE uses finite fields based Diffie-Hellman groups.
+ *
+ * The corresponding family type is ::psa_dh_family_t. In determining a
+ * specific group in the family ::psa_pake_bits_t values are interpreted in the
+ * exact same way as ::psa_key_bits_t would.
+ *
+ * Input and output during the operation can involve group elements and scalar
+ * values:
+ * -# The format for group elements is the same as for public keys on the
+ *  specific group would be. For more information, consult the documentation of
+ *  psa_export_public_key().
+ * -# The format for scalars is the same as for private keys on the specific
+ *  group would be. For more information, consult the documentation of
+ *  psa_export_key().
+ */
+#define PSA_PAKE_PRIMITIVE_TYPE_FIELD_DH       ((psa_pake_primitive_type_t)0x02)
+
+/** Construct a PAKE primitive from type, family and bitsize.
+ *
+ * \param type          The type of the primitive
+ *                      (value of type ::psa_pake_primitive_type_t).
+ * \param family        The family of the primitive
+ *                      (the type and interpretation of this parameter depends
+ *                      on \p type, for more information consult the
+ *                      documentation of individual ::psa_pake_primitive_type_t
+ *                      constants).
+ * \param bits          The bitwise of the primitive
```

Comment:
```suggestion
 * \param bits          Value of type ::psa_pake_bits_t. The interpretation
```

Reply:
This was indeed a typo, but I think that it is better to fix it than to delete the whole line.